src/lib/server.ts

```diff
@@ -419,6 +433,19 @@ export class ZwavejsServer extends EventEmitter {
     await once(this.server, "listening");
     this.emit("listening");
     this.logger.info(`ZwaveJS server listening on ${localEndpointString}`);
+    this.responder = getResponder();
+    this.service = this.responder.createService({
+      name: this.driver.controller.homeId!.toString(),
```

Comment:
Can we make this opt-in/opt-out ?

Reply:
yep, which do you prefer? opt in? we can add a new flag for it

Reply:
what's your thought behind that btw? given we are hard open to the network, I can't imagine it's a security concern

Reply:
It's causing extra traffic that is not needed in certain circumstances (ie the HA add-on).

We can make it opt-out.